src/base/tristatebool.h

```diff
@@ -36,15 +36,26 @@ class TriStateBool
     static const TriStateBool False;
     static const TriStateBool True;
 
-    TriStateBool() = default;
-    TriStateBool(const TriStateBool &other) = default;
+    constexpr TriStateBool() = default;
+    constexpr TriStateBool(const TriStateBool &other) = default;
+    explicit TriStateBool(const int value);  // add constexpr when using C++14
```

Comment:
Please don't add const to "value" here.

Reply:
pardon, was a leftover, will clean it up.